nucypher/policy/collections.py

```diff
@@ -163,6 +167,29 @@ def public_id(self):
         _id = keccak_digest(bytes(self._verifying_key) + bytes(self._hrac)).hex()
         return _id
 
+    def __serialize(self):
+        """
+        WAIT! This is probably not what you want. In 9/10 times, you'll
```

Comment:
It's not clear what's happening here to me - What are the differences between this and the other serilizaers?

Reply:
The "normal" serializer also encrypts for Bob (yeah, very convoluted logic). This one serializes a plaintext TreasureMap

Reply:
Yes, essentially, when you call `bytes(...)` on a `TreasureMap`, it assumes you want it encrypted for Bob. Attempting to change this causes massive ripple effects.